docs/hosted-orchestration/building-images.md

```diff
@@ -0,0 +1,168 @@
+---
+id: building-images
+title: Building Docker Images
+sidebar_label: Building Images
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This document describes how to create and use custom Docker images with Sauce Labs.
+
+## Overview
+Sauce Labs supports Docker, providing you with a powerful way to orchestrate tests. In order to use Hosted Orchestration you will need to package your test code and all of its dependencies as a Docker Image and publish it to a Docker container regisry.
+
+## System Requirements
+
+You can run build images locally via Docker ([Installation Requirements](https://docs.docker.com/engine/install/#supported-platforms)) or as part of your CI process. System requirements vary depending on your intention so the following reference serves as a general guide:
```

Comment:
"You can run build images" --> "You can build images"

I would also say "build and push". Technically, it's redundant because Docker always let's you do both.